src/commonMain/resources/index.html

```diff
@@ -2,6 +2,10 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
+
```

Comment:
This is what actually fixed the icons. Documented here:

https://material-ui.com/components/icons